Specified the OS requirements (#2915)
metabrainz/musicbrainz-server: README.md

General Prerequisites:

> * a Unix-like operating system (e.g. Debian GNU/Linux or Ubuntu)

Reviewer: That would be nice but it is untrue.

For example, it is known to **not** fully work on FreeBSD, from MBS-8041.

The `INSTALL.md` file mentions that _Mac OS X will work just fine, if you're prepared to potentially jump through some hoops_, which @mwiencek is doing for development, but it is not fully covered by the instructions.

However Ubuntu is the only platform that we use for production (so it is guaranteed to be compatible), that we recommend for mirrors and development setup through [our Docker Compose project](https://github.com/metabrainz/musicbrainz-docker), and that we provide full instructions for installing from source (which is not even recommended from [MusicBrainz Server / Setup](https://musicbrainz.org/doc/MusicBrainz_Server/Setup) doc).

At best, we can say that it is very likely to work on other Linux distributions, and possibly on other Unix-like systems, if you are able to help yourself.

Author: None of the Unix-like operating systems that run on my servers support Docker. However, *in general*, Ubuntu and Debian are not “strict requirements” either. How could this be written so that it’s *true*?

Reviewer: We don’t support other platforms because we don’t all use these, so we are **not** be able to check that our patches would not break the compatibility with these other platforms.

For example, I cannot assert that it will work on Solaris. However patches that help with improving the overall portability are welcome. We can also accept separate platform-specific instructions such as a `INSTALL_SOLARIS.md` file for example.

For now, I would suggest to mention other platforms very cautiously:

```suggestion
* Ubuntu/Debian (or some other Unix-like operating system at your own risks)
```

Author: Good point.
